spec/mailers/provider_mailer_spec.rb

```diff
@@ -3,214 +3,112 @@
-    before do
-      @mail = mailer.application_submitted(provider_user, application_choice)
-    end
-
-    it 'sends an email with the correct subject' do
-      expect(@mail.subject).to include(
-        t('provider_application_submitted.email.subject',
-          course_name_and_code: application_choice.course.name_and_code),
-      )
-    end
-
-    it 'addresses the provider user by name' do
-      expect(@mail.body.encoded).to include("Dear #{provider_user.full_name}")
-    end
-
-    it 'includes the candidate name' do
-      expect(@mail.body.encoded).to include("#{application_choice.application_form.full_name} submitted an application for")
-    end
-
-    it 'includes the course details' do
-      expect(@mail.body.encoded).to include(application_choice.course.name)
-      expect(@mail.body.encoded).to include(application_choice.course.code)
-    end
+  before do
+    provider = create(:provider, :with_signed_agreement, code: 'ABC')
+    course = create(:course, provider: provider, name: 'Computer Science', code: '6IND')
+    site = create(:site, provider: provider)
+    @course_option = create(:course_option, course: course, site: site)
+    @application_choice = create(:submitted_application_choice,
+                                 course_option: @course_option,
+                                 reject_by_default_at: Time.zone.now + 40.days,
+                                 reject_by_default_days: 123,
+                                 application_form:
+                                 create(
+                                   :completed_application_form,
+                                   first_name: 'Harry',
+                                   last_name: 'Potter',
+                                  ))
+    @provider_user = @application_choice.provider.provider_users.first
+    @provider_user.update(first_name: 'Johny', last_name: 'English')
+    @application_choice.application_form.update(submitted_at: Time.zone.now - 5.days)
+  end
 
-    it 'includes the reject by default days' do
-      expect(@mail.body.encoded).to include('after 123 working days')
-    end
+  describe 'Send account created email' do
+    it_behaves_like('a provider mail with subject and content', :account_created,
+                    I18n.t!('provider_account_created.email.subject'),
+                    'provider name' => "Dear Johny English\r\n",
+                    'sign in path' => '/provider/sign-in')
+  end
 
-    it 'includes a link to the application' do
-      expect(@mail.body.encoded).to include(provider_interface_application_choice_url(application_choice))
-    end
+  describe 'Send application submitted email' do
+    it_behaves_like('a provider mail with subject and content', :application_submitted,
+                    I18n.t!('provider_application_submitted.email.subject',
+                            course_name_and_code: 'Computer Science (6IND)'),
+                    'provider name' => "Dear Johny English\r\n",
```

Comment:
Same here, do we need the `\r\n`?

Reply:
I'm pretty sure I had a failing test because of this, but now I tried and no, we don't need 👌